### PR TITLE
Convert CoreReaderFragment.java into kotlin

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes: #3035 
Parent Issue: #1826 

Converted CoreReaderFragment.java into kotlin

### CODE REVIEW

**Comment by kelson42** [comment]
@gouri-panda Can you please review?

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt [hunk 1]
```diff
@@ -0,0 +1,1700 @@
...
+  BaseFragment(),
+  WebViewCallback,
+  MenuClickListener,
+  FragmentActivityExtensions,
+  WebViewProvider {
+  protected val webViewList: MutableList<KiwixWebView> = ArrayList()
+  private val webUrlsProcessor = BehaviorProcessor.create<String>()
+
+  @JvmField @BindView(R2.id.toolbar)
+  var toolbar: Toolbar? = null
```
Comment (gouri-panda) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt:149]: Can we have two lines for project consistency?

**Review by gouri-panda — CHANGES_REQUESTED** [review]
@MohitMaliDeveloper It keeps failing almost all CI.

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt (hunk 1 above)
Comment (MohitMaliFtechiz) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt]: hi @gouri-panda  Done.

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliDeveloper It keeps failing almost all CI.

hi @gouri-panda ,
fixed CI failure.

**Review by gouri-panda — APPROVED** [review]
@MohitMaliFtechiz  Thanks for the hard work!

**Pull request merged**